### ISSUE TITLE
Clarify License

### ISSUE THREAD

**Issue by davidak** [TilCreator/NixOwO#1]
Very nice image!

Where did you get the license from?

This seem to be the original post: https://www.reddit.com/r/linuxmasterrace/comments/lzdwl4/nixowos/
It does not have a license.

It is referenced here as to have CC BY: https://github.com/TheDarkBug/uwufetch/blob/main/res/COPYRIGHT.md#nixos-nix-snowflake

With a CC license that has BY, you have to name the original author "[ant-arctica](https://www.reddit.com/user/ant-arctica/)".

When the license is clarified, please add a LICENSE file to the repo.

**Comment by TilCreator** [comment]
I had a second look at the copyright a license situation. I don't see it as necessary to name the artist as long as there is a link (or other direct reference) to the original work where all of that information can be found.
I used the CC-BY-SA here because the NixOS website has that license.

To make it more clear I will add the section from the COPYRIGHT.md to the readme of this repo.

**Comment by TilCreator** [comment]
@davidak Do you consider this issue solved?

**Comment by davidak** [comment]
tl;dr yes, it's solved. thanks!

>I don't see it as necessary to name the artist as long as there is a link (or other direct reference) to the original work where all of that information can be found.

The site says:

>You must give [appropriate credit](https://creativecommons.org/licenses/by/4.0/#)

And as explanation:

>If supplied, you must provide the name of the creator and attribution parties, a copyright notice, a license notice, a disclaimer notice, and a link to the material.

That's pretty clear i think.

>I used the CC-BY-SA here because the NixOS website has that license.

That is a good license for a final content piece, so modifications have to have the same license, but could be too limiting here. For example i don't think it would be allowed to use this logo in NixOS, because it's not CC BY-SA, but that would be fun.

But you have CC BY now, so everything is fine.

>To make it more clear I will add the section from the COPYRIGHT.md to the readme of this repo.

Looks very good!

It's still unclear what "@timbertson" means. Is it a Twitter handle or GitHub user? But i think it's fine unless they complain.

(It's actually both)

https://github.com/timbertson
https://twitter.com/timbertson

uwu

**Comment by timbertson** [comment]
I don't know what this repo does but the logo's cute 😀

**Comment by davidak** [comment]
@timbertson it's just an extra cute version of the logo (as vector graphic, SVG). Someone else did draw it.
